service/src/test/java/uk/gov/hmcts/reform/fpl/service/DraftCMOServiceTest.java

```diff
@@ -94,25 +93,19 @@ void shouldNotReturnHearingDatesWhenHearingDateIsInThePast() {
         hearingDetails = createHearingBookingsFromInitialDate(time.now().minusDays(10));
         caseManagementOrder = CaseManagementOrder.builder().build();
 
-        Map<String, Object> data = draftCMOService.extractCaseManagementOrderVariables(
-            caseManagementOrder, hearingDetails);
+        DynamicList data = service.getHearingDateDynamicList(hearingDetails, caseManagementOrder);
```

Comment:
This refactoring doesn't have anything to do with the ticket, but it's fine I don't mind.

Reply:
so this isn't a full implementation of 1483, I just keep getting hit with horrific merge conflicts when I'm working on large tickets. So I am trying a little PR approach. As we have removed the extractCaseManagementOrderVariables method from draftCMO service, I have refactored what the tests were testing (getHearingDateDynamicList method) so that this functionality is still tested. To be revisited in further PRs